src/components/slop-fest-modal.jsx

```diff
@@ -23,6 +22,7 @@ export function SlopFestModal({ buttonTitle, location, fest }) {
       setValue("startDate", festStart);
       setValue("endDate", festEnd);
       setValue("name", fest?.data?.fest?.name);
+      setValue("invitees", inviteesBefore);
```

Comment:
Invitees wasn't prefilling on edit-slop page. So I added this quick fix so I could properly test if the combobox was still allowing doubles.